packages/default/scss/scrollview/_theme.scss

```
@@ -9,48 +9,73 @@
             $scrollview-bg,
             $scrollview-border
         );
+        -webkit-tap-highlight-color: $rgba-transparent;
 
-        // refactor when https://github.com/telerik/kendo-angular-scrollview/issues/130 is done
-        .k-scrollview-pageable > li.k-button,
-        .k-scrollview-nav > li.k-link {
-            @include fill(
-                $bg: $scrollview-pagebutton-bg,
-                $border: $scrollview-pagebutton-border
-            );
-
-            box-shadow: $scrollview-pagebutton-shadow;
-            background-clip: padding-box;
+        &.k-scrollview-dark {
+            kendo-scrollview-pager,
+            .k-scrollview-nav-wrap {
+                background-color: $scrollview-dark-bg;
+            }
+            .k-scrollview-next,
+            .k-scrollview-prev {
+                color: $scrollview-dark-bg;
 
-            &.k-primary {
-                @include fill(
-                    $bg: $scrollview-pagebutton-primary-bg,
-                    $border: $scrollview-pagebutton-primary-border
-                );
             }
         }
 
-        .k-scrollview-elements {
-            color: $scrollview-navigation-color;
+        &.k-scrollview-light {
+            kendo-scrollview-pager,
+            .k-scrollview-nav-wrap {
+                background-color: $scrollview-light-bg;
+            }
+
         }
+    }
+
+    .k-scrollview-next,
+    .k-scrollview-prev {
+        color: $scrollview-navigation-color;
+        background-color: $scrollview-navigation-bg;
+        text-shadow: $scrollview-navigation-icon-shadow;
+        opacity: $scrollview-navigation-default-opacity;
+        outline-width: 0;
+        -webkit-tap-highlight-color: $scrollview-arrow-tap-highlight-color;
 
-        .k-scrollview-next,
-        .k-scrollview-prev {
-            color: inherit;
-            background: $scrollview-navigation-bg;
-            text-shadow: $scrollview-navigation-icon-shadow;
-            opacity: $scrollview-navigation-default-opacity;
-            outline-width: 0;
-
-            &:hover {
-                color: $scrollview-navigation-color;
-                opacity: $scrollview-navigation-hover-opacity;
-
-                span::before {
-                    background-color: $scrollview-navigation-hover-span-bg;
-                }
+        &:focus,
+        &.k-state-focused {
+            color: $scrollview-navigation-color;
+            opacity: $scrollview-navigation-hover-opacity;
+            .k-icon::before {
```

```suggestion

            .k-icon::before {
```

leave a space for legibility.